CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Changelog
 
+## [1.50](https://github.com/brave/brave-ios/releases/tag/v1.50)
+
+### Web3
+
+- Added support for opening ipns:// scheme URL from other apps. (#[7227](https://github.com/brave/brave-ios/issues/7227))
+- Added support for Unstoppable Domains Navigation. (#[7089](https://github.com/brave/brave-ios/issues/7089))
+- Added support for IPFS Navigation in omnibox. (#[7013](https://github.com/brave/brave-ios/issues/7013))
+- Added validation for Brave Wallet Send to disable the "Send" button when address is missing checksum.  (#[7136](https://github.com/brave/brave-ios/issues/7136))
+- Fixed favicons displaying very small on lock screen widget. (#[7124](https://github.com/brave/brave-ios/issues/7124))
+- Fixed connecting wallet issue on poap.fun dApp. (#[5727](https://github.com/brave/brave-ios/issues/5727))
```

Comment:
Changed label of this one. The actual issue is going to be fixed on 1.51 so please remove this one from the list.